templates/news-detail/news-detail.js

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable function-paren-newline, object-curly-newline */
+import { div, h2, a, img } from '../../scripts/dom-helpers.js';
+import { getMetadata } from '../../scripts/aem.js';
+import { breadcrumbs } from '../../scripts/breadcrumbs.js';
+
+export default function decorate(doc) {
+  const $main = doc.querySelector('main');
+  const $picture = $main.querySelector('picture');
+  const $hero = div({ class: 'hero' },
+    $picture,
+  );
+
+  const $content = $main.querySelector('.default-content-wrapper');
+  const teaserTitle = getMetadata('teaser-title');
```

Comment:
I think, we can use `title` property for `teaser-title`. Same for `description` and `teaser-description`
Will help us reduce the number of metadata fields.

Reply:
We will need confirmation from the customer - they are different